File: app/src/main/java/com/angogasapps/myfamily/ui/customview/message_recycle_view/holders/TextMessageHolder.java
package com.angogasapps.myfamily.ui.customview.message_recycle_view.holders;

import android.view.View;
import android.widget.TextView;

import androidx.annotation.NonNull;

import com.angogasapps.myfamily.R;
import com.angogasapps.myfamily.databinding.TextMessageHolderBinding;

public class TextMessageHolder extends AppBaseViewHolder{

    public TextMessageHolder(@NonNull View itemView) {
        super(itemView);
    }

    @Override
    protected void initLeftFields() {
        super.initLeftFields();
        ((TextView)view.findViewById(R.id.leftMessageText)).setText(value);
    }

    @Override
    protected void initRightFields() {
        super.initRightFields();
        ((TextView)view.findViewById(R.id.rightMessageText)).setText(value);
    }
}

File: app/src/main/java/com/angogasapps/myfamily/ui/screens/chat/holders/TextMessageHolder.kt
package com.angogasapps.myfamily.ui.screens.chat.holders

import android.view.View
import com.angogasapps.myfamily.R
import android.widget.TextView

class TextMessageHolder(view: View) : AppBaseViewHolder(view) {

    override fun initLeftFields() {
        super.initLeftFields()
        (rootView.findViewById<View>(R.id.leftMessageText) as TextView).text = value
    }

    override fun initRightFields() {
        super.initRightFields()
        (rootView.findViewById<View>(R.id.rightMessageText) as TextView).text = value
    }
}